### PR TITLE
Rich Commands: Decompress Archive commands

### PR DESCRIPTION
**Pull request by QuaintMako**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Related to #11481

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [ ] Tested the changes for accessibility

### CODE REVIEW

**Comment by QuaintMako** [comment]
I am pretty close to finish those commands, but I have an issue with the text for the inner toolbar. For the `DecompressToChildFolderAction`, the label is evaluated once at the creation of the toolbar, and I cannot find a proper way to refresh it. No other command have the need to reevaluate its label yet. So we may not have the tools to do it yet?

**Comment by cinqmilleans** [comment]
Adding Mode=OneWay to the binding to the label in the xaml should work.

**Comment by QuaintMako** [comment]
> Adding Mode=OneWay to the binding to the label in the xaml should work.

Did the trick, thanks a lot!

File: src/Files.App/Views/ModernShellPage.xaml.cs [hunk 1]
```diff
@@ -190,12 +190,6 @@ private async void KeyboardAccelerator_Invoked(KeyboardAccelerator sender, Keybo
 
 			switch (c: ctrl, s: shift, a: alt, t: tabInstance, k: args.KeyboardAccelerator.Key)
 			{
-				case (true, false, false, true, VirtualKey.E): // ctrl + e, extract
```
[the remark this answers is not in the document]
Comment (QuaintMako) [reply src/Files.App/Views/ModernShellPage.xaml.cs:193]: Should I go ahead and add it to the documentation?
Another good find from the Rich Commands!

[the remark this answers is not in the document]
Comment (yaira2) [reply src/Files.App/Views/ModernShellPage.xaml.cs:193]: If you don't mind, thanks!

**Pull request merged**